### DISCUSSION TITLE
Overlap between paddle and ball using version 0.4.9

### DISCUSSION THREAD

**Discussion by 4lejandrito** [pmndrs/use-cannon#94] [General] [answered]
Hi! I've been playing with your ping pong example. I found that when using `use-cannon@0.4.9` the paddle and the ball overlap: 

<img width="456" alt="Screenshot 2020-07-03 at 18 29 54" src="https://user-images.githubusercontent.com/1937354/86486409-7b81d000-bd5b-11ea-9c6a-59893040bcab.png">

(Check out [my fork with just the version change](https://codesandbox.io/s/damp-darkness-yevny?file=/package.json)).

If I use `0.2.11` it works fine again.

I am new to 3D stuff so I have no clue why this is.

Thanks for such an awesome hook!

**Comment by drcmda** [comment, marked as answer]
My fault, I fixed the args prop of useBox. It doesn't take half Extents as before, but full length width height just like when you construct box geometry. I will update the sandboxes when I get to it
